airsenal/scripts/fill_fixture_table.py

```diff
@@ -87,12 +92,12 @@ def fill_fixtures_from_api(season, dbsession=session):
             raise ValueError(f"Can't find team(s) with id(s): {away_id}")
         if not update:
             dbsession.add(f)
-
     dbsession.commit()
-    return True
```

Comment:
Seems like this bool is never used anywhere

Reply:
It looks like there are a few places we `return True` (or False) without doing anything with them. They could all be removed as far as I'm concerned.